caddyhttp/httpserver/plugin.go

```diff
@@ -437,6 +437,7 @@ var directives = []string{
 	"hugo",        // github.com/hacdias/caddy-hugo
 	"mailout",     // github.com/SchumacherFM/mailout
 	"awslambda",   // github.com/coopernurse/caddy-awslambda
+	"maxrequestbody",
```

Comment:
Doesn't seem like anything other directives use underscore at the moment.

Also I will leave renaming last.